src/fastapi_poe/base.py

```diff
@@ -150,9 +150,9 @@ def __post_init__(self) -> None:
 
     async def post_message_attachment(
         self,
-        access_key: str,
```

Comment:
This will break all existing bots using this function and passing positional arguments. Unfortunately I'm not sure there's an easy solution that retains compatibility. One approach could be to make both the `access_key` and `message_id` arguments optional, then use `@typing.overload` to mark the variant with positional arguments optional. I can push a version of that if you like.

Reply:
Yeah.Making things backward compatible would be best. Thanks, Jelle!

Reply:
Done! Didn't test it against a live bot though.

Reply:
Users would be expected to pass all parameters to `post_message_attachment` as keywords, e.g. `await self.post_message_attachment(message_id=whatever, filename="some_file")`.